client-app/shared/loyalty/composables/usePointsOperations.ts

```diff
@@ -0,0 +1,96 @@
+import { PointsOperationsSearchParams, registerPointsOperation, searchMyPointsOperations } from "@core/api/graphql/loyalty";
+import { Logger } from "@core/utilities";
+import { InputRegisterPointsOperationType, PointsOperationType, UserType } from "@core/api/graphql/types";
+import { MaybeRef } from "@vueuse/core";
+import { computed, readonly, ref, Ref, shallowRef, unref } from "vue";
+
+const fallbackItemsPerPage = 20;
+
+const loading: Ref<boolean> = ref(true);
+const loadingMore: Ref<boolean> = ref(false);
+const pointsOperations: Ref<PointsOperationType[]> = shallowRef([]);
+const total: Ref<number> = ref(0);
+const pages: Ref<number> = ref(1);
+
+export default (options: { userId: MaybeRef<string>, storeId: MaybeRef<string> }) => {
+  const { userId, storeId } = options;
```

Comment:
See above comments about these variables. Maybe it's better to fetch them here from `constants.ts` or make this login inside queries/mutations as done on other samples.